qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -187,6 +178,24 @@ def __init__(
         # Push the schedules to the instruction schedule map.
         self.update_inst_map()
 
+    @property
+    def drive_freq(self):
+        """Property only here to deprecate `drive_freq`."""
+        warnings.warn(
+            "The variable `drive_freq` has been deprecated and will be removed after "
+            "Qiskit Experiments 0.6 is released."
+        )
+        return self._parameter_map.get(("drive_freq", (), None), None)
```

Comment:
```suggestion
    @property
    @deprecate_func(
        is_property=True,
        since="0.6",
        package_name="qiskit-experiments",
        additional_msg=(
            "This attribute is moved to :class:`.FixedFrequencyTransmon` basis gate library."
        ),
    )
    def drive_freq(self):
        """Parameter object for qubit drive frequency."""
        return self._parameter_map.get(("drive_freq", (), None), None)
```

You don't need to wait for #1186 to be merged. This PR is for updating the rest of old deprecation usage in our stack and new deprecation should use this decorator. Note that this decorator also automatically updates API doc, you don't need to say this is deprecated. Same for `meas_freq`. Apart from this, this PR looks good to go.

Reply:
Thanks!